src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -169,7 +171,17 @@ public static String composeRpcMethodHeaderSampleCode(
       Method method,
       List<MethodArgument> arguments,
       TypeNode clientType,
-      Map<String, ResourceName> resourceNames) {
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes) {
+    if (method.isPaged()) {
+      Field pagedItemField =
+          messageTypes.get(method.outputType().reference().name()).fields().get(0);
+      TypeNode pagedItemType =
```

Comment:
The pagedItemType does not necessarily meet this condition, please use `findAndUnwrapFirstRepeatedField` [here](https://github.com/googleapis/gapic-generator-java/blob/master/src/main/java/com/google/api/generator/gapic/model/Message.java#L58) instead.

Reply:
SG